test/src/tests/generated/body-array/interfaces.go

```diff
@@ -0,0 +1,89 @@
+package arraygroup
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+import (
```

Comment:
i thought we would do the interfaces in a subdir ?

Reply:
Yep working on that now.  It gets a little tricky in the codegen since some param/return types are defined in the parent package.